src/cljs/athens/events.cljs

```diff
@@ -248,18 +249,21 @@
     (update db :left-sidebar/open not)))
 
 
-(reg-event-db
+(reg-event-fx
   :right-sidebar/toggle
   [(interceptors/sentry-span "right-sidebar/toggle")]
-  (fn [db _]
-    (update db :right-sidebar/open not)))
+  (fn [{:keys [db]} _]
+    (let [closing? (:right-sidebar/open db)]
+      {:db       (update db :right-sidebar/open not)
+       :dispatch [:posthog/report-feature :right-sidebar (not closing?)]})))
```

Comment:
Why `(not closing?)`

Reply:
because when `closing?` is truthy, right sidebar is getting inactive.